.github/workflows/badges.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-20.04
     if: github.repository == 'commaai/openpilot'
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # pin@v3
```

Comment:
this seems excessive for GitHub's own actions

Reply:
Regardless of where the Action comes from, it helps to pin Actions. Even if it is from Actions themselves :smiley: